migrator/migrations/m_197_to_m_198_add_oidc_claim_mappings/migration_test.go

```diff
@@ -0,0 +1,311 @@
+//go:build sql_integration
+
+package m197tom198
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/migrator/migrations/m_197_to_m_198_add_oidc_claim_mappings/schema"
+	"github.com/stackrox/rox/migrator/migrations/m_197_to_m_198_add_oidc_claim_mappings/store/authproviders"
+	"github.com/stackrox/rox/migrator/migrations/m_197_to_m_198_add_oidc_claim_mappings/store/groups"
+	pghelper "github.com/stackrox/rox/migrator/migrations/postgreshelper"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/declarativeconfig"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stretchr/testify/suite"
+)
+
+var ctx = sac.WithAllAccess(context.Background())
+
+type apMigrationTestSuite struct {
+	suite.Suite
+
+	db *pghelper.TestPostgres
+}
+
+func TestMigration(t *testing.T) {
+	suite.Run(t, new(apMigrationTestSuite))
+}
+
+func (s *apMigrationTestSuite) SetupSuite() {
+	s.db = pghelper.ForT(s.T(), false)
+	pgutils.CreateTableFromModel(ctx, s.db.GetGormDB(), schema.CreateTableAuthProvidersStmt)
+	pgutils.CreateTableFromModel(ctx, s.db.GetGormDB(), schema.CreateTableGroupsStmt)
+}
+
+func (s *apMigrationTestSuite) TearDownSuite() {
+	s.db.Teardown(s.T())
+}
+
+var unmigratedAuthProviders = []*storage.AuthProvider{
+	{
+		Id:   declarativeconfig.NewDeclarativeAuthProviderUUID("no-claims").String(),
```

Comment:
Why use the declarative config UUID stuff if you could just use `uuid.NewV5FromNonUUID("test", "some-name")`?

While roxvet is nice to use and apparently ignores the test files for import restrictions, I don't think we want to import this here solely for the purpose of UUID generation.